code.py

```diff
@@ -1,9 +1,17 @@
 from purple.core import Core
 
-from purple.keyboard.mist.keyboard import keys
-from purple.keyboard.mist.layout.soar_left import Layout
+#from purple.keyboard.mist.keyboard import keys
```

Comment:
As a general comment, I'm going to be renaming this file to **example code.py** and requesting that **code.py** not be included in pull requests, since everyone will be updating it to test their keyboard anyways.

Reply:
Cool, yeah, I wasn't sure how you'd want to do that.  Maybe rename it to matrix_example_code.py.